docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+    web:
```

Comment:
next services to add:

- `goaws` for SQS simulation
- separate `php-cli` for the long-running process